plugin/evm/header/extra.go

```diff
@@ -8,10 +8,15 @@ import (
 	"fmt"
 
 	"github.com/ava-labs/coreth/core/types"
-	"github.com/ava-labs/coreth/params"
 	"github.com/ava-labs/coreth/params/extras"
 )
 
+const (
+	// Note: MaximumExtraDataSize has been reduced to 32 in Geth, but is kept the same in Coreth for
+	// backwards compatibility.
+	MaximumExtraDataSize uint64 = 64 // Maximum size extra data may be after Genesis.
```

Comment:
```suggestion
	MaximumExtraDataSize = 64 // Maximum size extra data may be after Genesis.
```

I think generally it is better if we let the compiler infer the types. This would allow us to remove the explicit cast lower in the file.